libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/MapboxMetricsReporter.kt

```diff
@@ -57,7 +81,8 @@ object MapboxMetricsReporter : MetricsReporter {
      */
     @JvmStatic
     fun toggleLogging(isDebugLoggingEnabled: Boolean) {
-        mapboxTelemetry.updateDebugLoggingEnabled(isDebugLoggingEnabled)
+        // todo need support on core side
```

Comment:
Is there a ticket for this?

Reply:
@tarigo @brunoabinader @tatiana-yan could you help with it? Did this API migrate to core telemetry?

Reply:
There are two functions which we decided to deprecate, we do not think we need to expose them,
it's `setDebugLoggingEnabled` and `setSessionIdRotationInterval`

maps-android also deprecated it
for ios this functions never exist

do you think it's possible to deprecate it for navigation-android also?

Reply:
Do you need it for some internal usage or clients use it?
As I understand, the only logs which were enabled with this setting are 
[1](https://github.com/mapbox/mapbox-events-android/blob/d73ca293b194376516997b73c5139b28430be671/libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryClient.java#L101) and [2](https://github.com/mapbox/mapbox-events-android/blob/d73ca293b194376516997b73c5139b28430be671/libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryClient.java#L154)

It's possible to get this information for debug from [HttpInterceptor](https://github.com/mapbox/mapbox-sdk-common/blob/64638e9e472634d3d06db8930855a4449e69c60d/components/network/generated/java/com/mapbox/common/HttpServiceInterceptorInterface.java), see [commit](https://github.com/mapbox/1tap-android/commit/86fd5edeae7b9d97a667c2a7bdd7d1c8d39fb4c7) for reference

Reply:
I think it's possible to introduce some setting and manage it with `SettingsServiceFactory` but now we do not have such setting

Reply:
how is it possible to log the same things with HttpInterceptor if customers want it? 

I see [maps disabled it as well](https://github.com/mapbox/mapbox-maps-android/pull/1672/files#diff-33769bee9975a857459b416933afb97b8d9ce03ea47b6370eb7da79c0c090c77R130-R132). @pengdev are planning to support it or provide to customers another way to see these logs? 

cc @LukasPaczos

Reply:
There's no usage of that `setDebugLoggingEnabled` call in Maps SDK, it was decided to just remove it. cc @tobrun 
If customers is interested to see it, they can either use http interceptor or use network inspector of the IDE.

Reply:
I'm not sure if we have customers who uses it (@LukasPaczos maybe you know?). We can align this with maps (the method is deprecated and doing noting).

Reply:
Make no-op and deprecate sounds good to me. What debug logging entails is not quantifiable anyway, so there should be no risk.